app/controllers/users_controller.rb

```diff
@@ -49,9 +49,15 @@ def destroy
     end
   end
 
+  def set_current_deck
+    current_user.set_current_deck(params[:deck_id])
+    render "decks/index"
+  end
+
   private
 
   def user_params
-    params.require(:user).permit(:email, :password, :password_confirmation, :authentications_attributes)
+    params.require(:user).permit(:email, :password, :password_confirmation,
+				 :authentications_attributes, :current_deck_id)
```

Comment:
Align the parameters of a method call if they span more than one line.<br>Tab detected.